assets/out

```diff
@@ -79,7 +80,11 @@ tag() {
 }
 
 push_src_and_tags() {
-  git push --tags push-target HEAD:refs/heads/$branch
+  if [ "$force" = "true" ]; then
+    git push --tags push-target HEAD:refs/heads/$branch --force
+  else
+    git push --tags push-target HEAD:refs/heads/$branch
+  fi
 }
 
 push_tags() {
```

Comment:
Re: updating the `tag-only` push case -- would you ever need to force push if you're only updating tags?